impl/sql/domain/storage.go

```diff
@@ -41,25 +41,26 @@ CREATE TABLE IF NOT EXISTS Domains(
   MinInterval           BIGINT NOT NULL,
   MaxInterval           BIGINT NOT NULL,
   Deleted               INTEGER,
-  DeleteTimeMillis      BIGINT,
+  DeleteTimeSeconds      BIGINT,
```

Comment:
It was misnamed before. time.Unix() gives seconds, not Millis or Nanos.

Reply:
Up to you but I still think this will go wrong at some point when it gets compared with some nano value somewhere..

Reply:
I could use `UnixNanos`, but this leaves the zero time object undefined: 

https://godoc.org/time#Time.UnixNano:
UnixNano returns t as a Unix time, the number of nanoseconds elapsed since January 1, 1970 UTC. The result is undefined if the Unix time in nanoseconds cannot be represented by an int64 (a date before the year 1678 or after 2262). Note that this means the result of calling UnixNano on the zero Time is undefined.